src/device-registry/utils/create-event.js

```diff
@@ -544,31 +543,24 @@ const createEvent = {
                   }
                 });
 
-                const status = responseFromListEvents.status
-                  ? responseFromListEvents.status
-                  : "";
-
                 try {
                   callback({
                     success: true,
                     message: !isEmpty(missingDataMessage)
```

Comment:
just in case there are no measurements, get an appropriate message in the response body